diff -rc original/etc/palm/db/permissions/com.palm.phonecall modified/etc/palm/db/permissions/com.palm.phonecall
*** original/etc/palm/db/permissions/com.palm.phonecall	1970-01-01 01:00:00.000000000 +0100
--- modified/etc/palm/db/permissions/com.palm.phonecall      2015-12-31 10:59:40.411319592 +0100
***************
*** 0 ****
--- 1,10 ----
+ [
+     {
+         "type": "db.kind",
+         "object": "com.palm.phonecall:1",
+         "caller": "hu.tylla.*",
+         "operations": {
+             "read": "allow"
+         }
+     }
+ ]
